xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -159,10 +202,20 @@
         <Parameter Name="assemblyName" Type="System.Reflection.AssemblyName" />
       </Parameters>
       <Docs>
-        <param name="assemblyName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="assemblyName">The object that describes the assembly to be loaded.</param>
+        <summary>Abstract method to allow resolving and loading an assembly given its <see cref="T:System.Reflection.AssemblyName" />.</summary>
+        <returns>The loaded assembly or <see langword="null" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+ 
+## Remarks
+ The <xref:System.Runtime.Loader.AssemblyLoadContext.Load(System.Reflection.AssemblyName)> is an abstract method. During assembly resolution, it has the first chance to resolve the assembly name to a loaded assembly.  It can either load an assembly or return `null`.  If it returns `null`, the resolution process will continue. See <xref:System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyName(System.Reflection.AssemblyName)> for the full process description.
```

Comment:
I would remove the first sentence - it's just repeating what's already state in the summary.